concurrent_test.go

```diff
@@ -339,6 +361,10 @@ type writeWorker struct {
 	t *testing.T
 }
 
+func (w *writeWorker) operationType() OperationType {
+	return Write
+}
```

Comment:
Operation type should be property of operation, not taken for granted for whole history

Reply:
do not agree this.  Different workers process different operation types,e.g. writeWorker processes Write operation, and readWorker processes Read operation. 

I can add a property/field something like `opType OperationType` into both `readWorker` or `writeWorker` if you want.

Reply:
removed method `operationType()`